tests/fixed/UselessConditions.php

```diff
@@ -70,7 +78,7 @@ public function uselessIfConditionWithComplexCondition() : bool
     public function uselessIfConditionWithTernary() : bool
     {
         if ($this->isTrue()) {
-            return $this->isTrulyTrue();
+            return $this->isTrulyTrue() ? true : false;
```

Comment:
See https://github.com/slevomat/coding-standard/issues/604 - CS doesn't know the type of `$this->isTrulyTrue()` and in general it may produce broken code (the method would return non-bool). This is now still reported, but not auto-fixed.